catalog/dags/common/extensions.py

```diff
@@ -1,12 +1,32 @@
-EXTENSIONS = {
-    "image": {"jpg", "jpeg", "png", "gif", "bmp", "webp", "tiff", "tif", "svg"},
-    "audio": {"mp3", "ogg", "wav", "aiff", "flac", "wma", "mp4", "aac", "m4a", "m4b"},
+import mimetypes
+
+from common.constants import MEDIA_TYPES as SUPPORTED_TYPES
+
+
+FILETYPE_EQUIVALENTS = {
+    # Image extensions
+    "jpeg": "jpg",
+    "tif": "tiff",
+    # Audio extensions
+    "midi": "mid",
 }
 
+mimetypes.add_type("audio/midi", ".mid")
+mimetypes.add_type("audio/midi", ".midi")
+mimetypes.add_type("audio/x-matroska", ".mka")
```

Comment:
Querying the API DB returns nothing.

<details>
<summary>query</summary>

```sql
SELECT * FROM audio WHERE filetype = 'mka' OR url ILIKE '%.mka' LIMIT 1;
```
</details>

I have yet to come across a track like this to have an example. Do you have reservations about it? It's included in Wikimedia's mediaType.

Reply:
[Matroska contrainers are not supported by Chrome, Firefox, or Safari](https://jellyfin.org/docs/general/clients/codec-support/#container-compatibility). We would need to extract the audio from the container and potentially convert it to support playing them. We should probably ignore these files and create an issue to follow up on adding some kind of support for them if we think it is valuable to include them. If there are currently 0 of them, then it's probably not worth the effort.

Reply:
Providing a wide range of formats should be valuable to users; even if they cannot reproduce them in Openverse right now, they should be able to download them from the source. The funny is we don't have any of this currently.

```sql
openledger=> SELECT COUNT(*) FROM audio WHERE filetype = 'mka' OR url ILIKE '%.mka';
 count 
-------
     0
(1 row)
```

So, for now, we're safe. We could add a check in some DAG to alert if this type is detected.

Reply:
I agree with providing all available formats on the API level. Since it is an audio, we should add a check for this format to the Nuxt app, to display a message similar to what we currently do _for ogg on iOS_, IIRC, that this format cannot be played on the user's device. If we ever get any media with this format 😆